src/components/scenes/Staking/StakeOverviewScene.tsx

```diff
@@ -141,6 +142,11 @@ const StakeOverviewSceneComponent = (props: Props) => {
           stakeProviderInfo={stakePolicy.stakeProviderInfo}
         />
       </View>
+      {stakePosition == null ? (
+        <View style={styles.shimmer}>
+          <Shimmer isShown />
+        </View>
+      ) : null}
```

Comment:
This would look better than the current behavior but the hardcoded height is not ideal since there could be one or two items in the list, maybe even 3? Since the list has an unknown number I'd prefer if we moved the fetchStakePosition into the previous scene and pass the allocations into this scene as props.